tensorflow/python/keras/layers/core.py

```diff
@@ -130,6 +130,12 @@ class Dropout(Layer):
   Dropout consists in randomly setting
   a fraction `rate` of input units to 0 at each update during training time,
   which helps prevent overfitting.
+  
+  Usage Example:
+     import tensorflow as tf
```

Comment:
You can get rid of this import statement.

Reply:
Thanks , @RichardXiao13. I've removed it.